src/appshell/iappshellconfiguration.h

```diff
@@ -31,8 +31,19 @@ class IAppShellConfiguration : MODULE_EXPORT_INTERFACE
     virtual ~IAppShellConfiguration() = default;
 
     virtual bool isAppUpdatable() const = 0;
+    virtual bool isFullScreenAvailable() const = 0;
 
     virtual ValCh<QStringList> recentScoreList() const = 0;
+
+    virtual ValCh<bool> isPalettePanelVisible() const = 0;
```

Comment:
I think, better to implement this using only two method:

Channel<PanelKey> pannelVisibilityChanged()
bool isPannelVisible(PanelKey key)

PanelKey {
InstrumentsPanel,
StatusBar,
Navigator,
....
}

Reply:
thanks, fixed